Gopkg.toml

```diff
@@ -96,6 +97,11 @@ required = [
   name = "knative.dev/pkg"
   branch = "master"
 
+[[override]]
+  name = "knative.dev/eventing-contrib"
+  source = "github.com/n3wscott/eventing-contrib"
```

Comment:
we don't need your bitcoin mining...  🙄

Reply:
it was broken in eventing contrib until a PR went in.